tests/test_TFNetworkLayer.py

```diff
@@ -3896,7 +3912,7 @@ def test_TransposedConvLayer_2d_simple():
     })
     out = net.get_default_output_layer().output.copy_as_batch_feature_major()
     assert out.batch_shape == (None, 13, None, 2)
-    assert_equal(out.get_dim_tag(2), net.extern_data.get_default_input_data().get_time_dim_tag())
+    assert out.dim_tags[2].is_equal(net.extern_data.get_default_input_data().get_time_dim_tag(), derived_matches=True)
```

Comment:
`derived_matches=True` should also not be needed here. You have filter size 1 and strides 1 in the T axis, and the split dims before also should leave it like it is.